tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -1058,3 +1068,40 @@ export const checkThatFileVersionIsNotAvailable = async (
   await sidebar.openPanel({ page, name: 'versions' })
   await expect(page.locator(fileVersionSidebar)).toHaveText('No Versions available for this file')
 }
+
+export interface changePageArgs {
+  page: Page
+  pageNumber: string
+}
+export const changePagePersonalSpace = async (args: changePageArgs): Promise<void> => {
+  const { page, pageNumber } = args
+  await page.keyboard.down('End')
+  const pageNumberSelector = util.format(listItemPageSelector, pageNumber)
+  await expect(page.locator(pageNumberSelector)).toBeVisible()
+  await page.locator(pageNumberSelector).click()
+}
+
+export interface changeItemsPerPageArgs {
+  page: Page
+  itemsPerPage: string
+}
+export const changeItemsPerPage = async (args: changeItemsPerPageArgs): Promise<void> => {
+  const { page, itemsPerPage } = args
+  await page.locator(filesViewOptionButton).click()
+  await page.locator(itemsPerPageDropDownSelector).click()
+  await page.locator(util.format(itemsPerPageDropDownOptionSelector, itemsPerPage)).click()
+  //close the files view option view
+  await page.locator(filesViewOptionButton).click()
+}
+
+export const getFileListFooterText = async ({ page }): Promise<string> => {
+  return page.locator(footerTextSelector).textContent()
```

Comment:
we do not need `await` here?

Reply:
not really we return the promise, the function that requires the value returned by this will await.